arrow/math.py

```diff
@@ -27,7 +27,7 @@ def multichoose(n, k):
 
     combinations = 1.0
 
-    for i in xrange(n.size):
+    for i in moves.range(n.size):
```

Comment:
Interesting, so this is the new way to call `xrange`. Guess we'll be doing this wcEcoli soon : )

Reply:
Yes. `six.moves` is the workaround for Python 3 providing incompatible names.

It seemed best to `from six import moves` then call `moves.range()`, but we could consider alternatives like `from six.moves import range6`, or just `import six`, or something.

Reply:
Correction: ``from six.moves import range as range6`.